### DISCUSSION TITLE
Github Student application rejected

### DISCUSSION THREAD

**Discussion by AbhinavUpenn** [community/community#137991] [GitHub Education]
### Hi everyone,

Heyy Everyone! I've been trying to set up the github student pack to access the copilot for a couple weeks now but my application keeps getting rejected. (I've used my student ID in the proof on enrollment) 

Any idea what's causing this?
![image](https://github.com/user-attachments/assets/7229b97e-485f-4b66-970b-49bf42d38ee6)

Appreciate the help!

**Comment by SAYAF-14** [comment]
Response #1

**Comment by Injamul2k24** [comment]
In two factor authentication you need to follow this doc :
 https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication


Here the billing section fill this
![{42CD15DD-A581-4C3C-943B-D02AAC4EFB8A}](https://github.com/user-attachments/assets/4a174187-d614-47fc-9f22-f1a3c0ad5af1)

**Comment by sudolifeagain** [comment]
same here. I have done setting my 2FA settings but submisson keeps rejected.

**Comment by meower1** [comment]
Hi if ur student id doesn't contain an expiration date it'll get rejected. if thats ur case try using another document for proof

**Comment by aivo9** [comment]
Captain here!

I had the same issue, but I just got approved.

Got rejected multiple times. Time after time. Made few perceptions about how it works, and finally succeeded!

1. Put everything in order in your account settings. Just as the instructions advice you to do. Name, emails, bios, 2 phase authentication, etc.
2. Try to photograph different kind of official school documents with a bad webcam and get rejected multiple times over and over again.
3. Get frustrated.
4. Realize that there is no human approving your entries but artificial intelligence instead "reading" words from the pictures you send.
5. Improvise, adapt, overcome.
6. Open the notebook of your mobile phone and type:

[Your University of Fancy Science]

Student: [insert your name here]
School: [insert your school name (same as in the title) here]
Education: [insert the name of your education here] (I had to add ",Virtual learning" to it cuz I'm not on campus right now, and it got mad)
Study right: [insert the day when you got your study right and the day it ends (in US format e.g. 1 January 2024 - 31 December 2028)]

7. So basically everything is true and you are not lying, but you just made the proof yourself on your mobile phone's notepad.
8. Take a screenshot. Edit the picture with your mobile phone and cut off all the excess space so that the text looks as big as possible in the screen.
9. Open the edited picture on your mobile phone and take the phone so close to your laptop's webcam so that all the words are clearly visible in the picture.
10. Take the photo and check that all the information is visible in the photo.
11. Submit your application.
12. Get approved few minutes later.
13. Not your fault the system of approval is so horrible and broken so feel no shame about cutting the corners.
14. Return the favor and go help others.
15. Enjoy!

Welcome to the community!

*flies away*
